src/pages/mesh/basic/create-mesh.md

```diff
@@ -316,3 +316,23 @@ On the API Mesh Details screen, you can use the tabs to view different segments
 To download a copy of your mesh file, click the **Download Mesh Schema** button at the top of the API Mesh Details screen.
 
 ![download mesh](../../_images/download-mesh-schema.png)
+
+### Mesh configuration
+
+On the **Active Config** tab, you can view the mesh configuration for the currently active mesh. If you have not successfully deployed a mesh, the **Config** tab displays the latest mesh configuration file that you uploaded.
```

Comment:
Do you need to make a similar statement (If you have not successfully deployed a mesh...) in the description of the `--active` flag?

Reply:
`No active deployment found for mesh. Check the details and try again or try without the --active flag. RequestId: ${global.requestId}`